src/tickit/adapters/io/tcp_io.py

```diff
@@ -20,9 +20,10 @@ class TcpIo(AdapterIo[CommandAdapter]):
     host: str
     port: int
 
-    def __init__(self, host: str, port: int) -> None:
+    def __init__(self, host: str, port: int, separator: str = None) -> None:
         self.host = host
         self.port = port
+        self.separator = separator
```

Comment:
Nit: I'm not sure the name `separator` is completely right, maybe `terminator`? Or just add a comment explaining what it does, it's not immediately obvious to me from the name (unlike `host` and `port`). Not bothered by that enough to block merge though.

Reply:
Thanks Callum, i'm not too sure about this name either, the reason I chose separator is because that's the name of the parameter in `StreamReader.readUntil`. I guess in this context it really is a terminator though. It probably makes sense to change it, i'll do that now